README.md

```diff
@@ -2,48 +2,49 @@
 
 [![Build Status](https://travis-ci.org/nemiah/phpFinTS.svg?branch=master)](https://travis-ci.org/nemiah/phpFinTS)
 
-:exclamation:**Note: The current developer version contains a new API in the `FinTsNew` class.
-If you are just starting to use this library, consider using that already and ignore the `FinTs` class.**
-If you want to (continue) using the old class, note that its constructor has changed, whereas the
-[Release 1.6](https://github.com/nemiah/phpFinTS/tree/1.6) still has the old constructor. :exclamation:
+:exclamation:**Note: Be sure to grab the latest version, currently from the `master` branch until we have a fresher
+release.**
 
 A PHP library implementing the following functions of the FinTS/HBCI protocol:
 
  * Get accounts
- * Get bank parameters
  * Get balance
```

Comment:
Das fehlt noch.

Reply:
Also `FinTs::getBpd()`?

Reply:
Meinte **Get Balance** als separate Action.

Reply:
Das ist doch der Saldo, oder? Siehe #255.

Reply:
Hm, ich glaube wir sollten das in GetBalance umbenennen.

Reply:
Oh ok, den PR #255 hab ich nicht mitgekommen. Ja sollte GetBalance heißen imho.